website/src/components/DraftEditorExample/index.js

```diff
@@ -6,111 +6,102 @@
  */
 
 import React from 'react';
-import Link from '@docusaurus/Link';
-import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
-import useBaseUrl from '@docusaurus/useBaseUrl';
 import {Editor, EditorState, RichUtils, getDefaultKeyBinding} from 'draft-js';
-import classnames from 'classnames';
-
-import Layout from '@theme/Layout';
 
 import './css/example.css';
 import './css/draft.css';
 import './css/rich-editor.css';
 
-class RichEditorExample extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {editorState: EditorState.createEmpty()};
-
-    this.focus = () => this.editor.focus();
-    this.onChange = editorState => this.setState({editorState});
-
-    this.handleKeyCommand = this._handleKeyCommand.bind(this);
-    this.mapKeyToEditorCommand = this._mapKeyToEditorCommand.bind(this);
-    this.toggleBlockType = this._toggleBlockType.bind(this);
-    this.toggleInlineStyle = this._toggleInlineStyle.bind(this);
-  }
-
-  _handleKeyCommand(command, editorState) {
-    const newState = RichUtils.handleKeyCommand(editorState, command);
-    if (newState) {
-      this.onChange(newState);
-      return true;
-    }
-    return false;
-  }
+const {useState, useRef, useCallback} = React;
 
-  _mapKeyToEditorCommand(e) {
-    switch (e.keyCode) {
-      case 9: // TAB
-        const newEditorState = RichUtils.onTab(
-          e,
-          this.state.editorState,
-          4 /* maxDepth */,
-        );
-        if (newEditorState !== this.state.editorState) {
-          this.onChange(newEditorState);
-        }
-        return;
-    }
-    return getDefaultKeyBinding(e);
-  }
+function RichEditorExample(props) {
+  const [editorState, setEditorState] = useState(EditorState.createEmpty());
+  const editor = useRef(null);
 
-  _toggleBlockType(blockType) {
-    this.onChange(RichUtils.toggleBlockType(this.state.editorState, blockType));
-  }
+  const focus = () => {
```

Comment:
Nit: can you rename to `focusEditor`? This is shadowing the `window.focus()` method